user-console/src/main/java/org/pentaho/mantle/client/admin/UserRolesAdminPanel.java

```diff
@@ -120,12 +142,37 @@ private Widget createUsersPanel() {
     hSpacer.setWidth( "100%" );
     labelAndButtonsPanel.add( hSpacer );
     labelAndButtonsPanel.setCellWidth( hSpacer, "100%" );
-    newUserButton = new ThemeableImageButton( addButtonStyles, null, Messages.getString( "new" ) );
+    newUserButton = new ThemeableImageButton( addButtonStyles, null,"add new user"  ) {
+      @Override
+      public void onBrowserEvent(Event event) {
+        if ( event.getTypeInt() == Event.ONMOUSEUP || KeyCodes.KEY_ENTER == (char) event.getKeyCode() ) {
+          newUserOnKeyPress();
+        }
+      }
+    };
```

Comment:
As commented elsewhere, this could and imo should be handled by the `ThemeableImageButton` class, by forwarding the ENTER key to a click event.

Reply:
not necessarily, as it may change the behavior of the ThemeableImageButton itself

Reply:
These are benefic, wanted _breaking_ changes — the whole accessibility work is a breaking change to the user experience. The risk that these changes would cause any unwanted behavior look to be very small. All buttons should have default keyboard handling. If we later find a case where this is not desired wanted, we can consider adding an option to opt-out.

Reply:
Updated as you suggested